code/config/lib/Loot/Config/Yaml.hs

```diff
@@ -131,9 +131,9 @@ instance
     => OptionsToJsonList k ((l ::< us) ': is)
   where
     configToJsonList (ItemSub inner :& rest) =
-      ( fromString $ symbolVal (Proxy :: Proxy l)
-      , A.object $ configToJsonList inner
-      ) : configToJsonList rest
+      let label = fromString $ symbolVal (Proxy :: Proxy l)
```

Comment:
`fromString $ symbolVal (Proxy :: Proxy l)` is used in several places, I suggest extracting it into a function.